packages/backend/src/errorHandler.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction } from 'express'
```

Comment:
What do you think if we create a `middlewares` folder where we would have the global middlewares?

Reply:
mhm